notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-756_analysis.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<ul><li>Could you please add the units on the plots ? Force in Newtons and displacement in microns (or is it mm) ?</li><li>The x-axis is labelled "Date", it looks to me that it should be "Time"</li><li>Would also be good to add something to identify the dataset, the date for instance</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>